templates/privacy.html

```diff
@@ -160,8 +161,8 @@ <h2 class="has-text-white mt-2 mb-3 is-size-1 is-size-3-mobile has-text-weight-b
       </div>
       <div class="column is-5 ml-auto">
         <div class="mx-auto box p-6 has-background-light has-text-centered">
-          <a class="button is-link is-fullwidth" id="contribute-button"
-            href="mailto:help@openandroidinstaller.org">Write a mail</a>
+          <a class="button is-link is-fullwidth hover-transition" id="contribute-button"
+            href="mailto:hello@openandroidinstaller.org">Write an e-mail</a>
```

Comment:
There is a reason why this is `help@`. I want to know where people came from :)

Reply:
Ooops O.o

Reply:
fixed now

Reply:
No worries, It was written nowhere :) So you couldn't know